### ISSUE TITLE
Cannot open main menu or top panel after setting it's size to minimum.

### ISSUE THREAD

**Issue by saukymo** [budgie-desktop/budgie-desktop#708]
**Operating System**: Archlinux x86_64 Linux 4.8.4-1-ARCH
**Budgie version**: budgie-desktop 10.2.7

At first, when I opened a fullscreen game, the top panel donnot disappear. So I set the layout to minimum, and I can't open the main menu or top panel any more.

Is there any way to reset the default setting? 

Thx!

**Comment by JannikHa** [comment]
So the panel not hiding when a fullscreen application is present is already known and worked on #630 . Your other problem is pretty interesting, will try to reproduce it ASAP. For a quick solution though, just open Raveb with WindowsKey+N(/A) and navigate to the panel settings, thus you can easily change it back to the default size (33 i believe) or whatever size you want. :)

**Comment by saukymo** [comment]
@JannikHa Thanks a lot! It works.
In fact, I tried the second solution first. But it didn't work.  It shows that:
Message: libsession.vala:50: DESKTOP_AUTOSTART_ID not set, session registration may be broken (not running budgie-desktop?)

Then I tried the first solution. Opened Raven with `win + N`. And I just remove the top panel and add it back. And I found that I can't open main menu because all the applets were removed for some reason. So I added them back.

I'm sorry that I can't reproduce it again by setting the size and hope that you can find the real reason.
Thanks again!

**Comment by ikeydoherty** [comment]
TL;DR 3.22 is bricked.

**Comment by ikeydoherty** [comment]
We'll need to retest these issues with people who have patched 3.22 I think

**Comment by fossfreedom** [comment]
@ikeydoherty going to create a 17.04 ISO this weekend with GTK3.22 that has been patched.  Will let you know.

**Comment by ikeydoherty** [comment]
Cheers lad - will be interesting to see if this is just the 3.22 issue or there's a rabbithole involved

**Comment by fossfreedom** [comment]
@ikeydoherty This works just fine with the patched version of Mutter.  BTW - the patched mutter is now officially in Ubuntu 17.04.

Have tried reducing the panel size down to 16px (minimum size) - menu works fine.  Full screen works just fine.

**Comment by saukymo** [comment]
**Very very sorry for this isue.** I just figured out what happend. 
I was careless to remove the top panel and then add it back . Then all the applets were removed. So that the only way to open Raven is the shortcut `win + n` if I close it.
Maybe should keep the Raven sidebar contrl on the top panel or just keep the setting before removed it.

**Comment by ikeydoherty** [comment]
Sidebar control is an applet. It gets no special treatment. There are confirmation dialogs in the git version of Budgie now which prevent you from accidentally deleting your panel

**Comment by JoshStrobl** [comment]
Closing. If this issue persists using latest Budgie, feel free to re-open. We have since added a confirmation dialog and GTK 3.22 issues with popovers have been resolved as well.
